greenberry/utils.py

```diff
@@ -0,0 +1,75 @@
+def eval(str):         #Function to evaluate postfix expression
+    exp=list(str)
+    S=[]
+    i=0
+    while(i<len(exp)):
+        if(exp[i].isnumeric()):
+            S.append(exp[i])
+        else:
+            y=int(S[-1])
+            S.pop()
+            x=int(S[-1])
+            S.pop()
+            if(exp[i]=='+'):
+                S.append(x+y)
+            elif(exp[i]=='-'):
+                S.append(x-y)
+            elif(exp[i]=='*'):
+                S.append(x*y)
+            elif(exp[i]=='/'):
+                S.append(x/y)
+            else:
+                pass
+        i=i+1
+    print(S[0])
+
+def InfixCon():
```

Comment:
Okay i will make a commit but then inside InfixCon function i have to remove the input() operation because we are passing the string now as parameter .
Okay ?

Reply:
exactly @KushalVijay